cmd/flow-capture.go

```diff
@@ -0,0 +1,380 @@
+package cmd
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"net"
+	"net/textproto"
+	"os"
+	"os/exec"
+	"regexp"
+	"slices"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/eiannone/keyboard"
+	"github.com/netobserv/flowlogs-pipeline/pkg/config"
+	"github.com/rodaine/table"
+	"github.com/spf13/cobra"
+
+	"github.com/fatih/color"
+)
+
+var flowCmd = &cobra.Command{
+	Use:   "get-flows",
+	Short: "",
+	Long:  "",
+	Run:   runFlowCapture,
+}
+
+var (
+	regexes     = []string{}
+	flowsToShow = 40
+	raw         = "Raw"
+	standard    = "Standard"
+	pktDrop     = "PktDrop"
+	dns         = "DNS"
+	rtt         = "RTT"
+	display     = []string{pktDrop, dns, rtt}
+	lastFlows   = []config.GenericMap{}
+)
+
+func runFlowCapture(cmd *cobra.Command, args []string) {
+	go scanner()
+
+	wg.Add(len(addresses))
```

Comment:
wg and goroutines seem wrongly implemented here, we should have a `wg.Done()` call somewhere... It probably doesn't do what we'd expect here.
I'm trying to change & make sure it doesn't break things

Reply:
Ok thanks; feel free to update this PR directly if you want to

Reply:
done - if you want to review my 2 commits

Reply:
Code looks good ! I'm giving a last try before merging